src/character.cpp

```diff
@@ -2319,6 +2324,44 @@ void Character::mod_int_bonus( int nint )
     int_cur = std::max( 0, int_max + int_bonus );
 }
 
+Character::stat string_to_stat( const std::string &Stat )
```

Comment:
I recommend doing the (de)serialization via functions within the system introduced by #33262.